### PR TITLE
Structure

### PR DESCRIPTION
**Pull request by Draxeytina**
Add structure to the to-do list
Manage using Webpack

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1 @@
+import './style.css';
```
Comment (smunozmo) [inline src/index.js]: - [x] Your app structure is looking great! However, the goal of this part of the activity is to create the list dynamically, this means using JS to:

- Create an index.js file and set an array of some simple to do tasks (array of objects). Each task object should contain three keys:
i. description [string].
ii. completed [bool].
iii. index: [number].

- Write a function to iterate over the tasks array and populate an HTML list item element for each task.

- On page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task.

Please address this requirements as this is the goal of this activity.

**Review by smunozmo — CHANGES_REQUESTED** [review]
Hi @Draxeytina, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights 👏 

- Great work using the right GitHub Flow.
- Your README looks professional, well done!
- Commits are descriptive, good work.
- Linters are working properly, great job!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! 🤓 

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

⚠️  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (Draxeytina) [reply src/index.js]: Noted thank you! I implemented the same. I had thought it was all about the skeleton of the interface at first.

**Review by gemmen29 — APPROVED** [review]
Hi @Draxeytina, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Draxeytina** [comment]
Thank you @gemmen29 !

**Pull request merged**